src/utils/client.ts

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+import { Phone } from '../types/Phone';
+
+export async function getPhones() {
+  try {
+    const response: Phone[] = (await axios.get('./api/phones.json')).data;
+    console.log(response);
+  } catch (error) {
+    console.error(error);
```

Comment:
Helper function, need to return an error or change error state, not console.error.